docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -2,263 +2,243 @@
+features in comparison with the [`Estimator`](../api/qiskit/qiskit.primitives.Estimator) class, which implements the
+legacy Estimator V1 interface. (For more information, see the [V2 primitives migration guide](/api/migration-guides/v2-primitives).) Both can take circuits, observables, and parameters as inputs and return the locally
+computed expectation values.
 
-  provider = QiskitProvider()
-  backend = provider.get_backend('backend_name')
-  estimator = BackendEstimator(backend)
-  ```
+The following code prepares the inputs that will be used in the examples that follow. The expected input type for the
+observables is [`qiskit.quantum_info.SparsePauliOp`](../api/qiskit/qiskit.quantum_info.SparsePauliOp). Note that
+the circuit in the example is parametrized, but you can also run the Estimator on non-parametrized circuits.
 
-  There are some providers that implement primitives natively (see [the Qiskit Ecosystem page](https://qiskit.github.io/ecosystem/#primitives) for more details).
+<Admonition type="note">
+  Any circuit passed to an Estimator must **not** include any **measurements**.
 </Admonition>
-
-### Initialize observables
-
-The first step is to define the observables whose expected value you want to compute. Each observable can be any `BaseOperator`, like the operators from [`qiskit.quantum_info`](../api/qiskit/quantum_info).
-Among them it is preferable to use [`qiskit.quantum_info.SparsePauliOp`](../api/qiskit/qiskit.quantum_info.SparsePauliOp).
-
-```python
-from qiskit.quantum_info import SparsePauliOp
-
-observable = SparsePauliOp(["II", "XX", "YY", "ZZ"], coeffs=[1, 1, -1, 1])
-```
-
-### Initialize QuantumCircuit
-
-Next, create the [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit) for which you want to obtain the expected value.
-
 ```python
 from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter
+from qiskit.quantum_info import SparsePauliOp
 
+# circuit for which you want to obtain the expected value
 qc = QuantumCircuit(2)
+qc.ry(Parameter('theta'), 0)
 qc.h(0)
 qc.cx(0,1)
 qc.draw("mpl", style="iqp")
 ```
-
 ![Initial QuantumCircuit](/images/verify/simulate-with-qiskit-primitives/estimator-initialize.png "Initial QuantumCircuit")
 
-<Admonition type="note">
-  The [`qiskit.circuit.QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit) you pass to [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) must not include any measurements.
-</Admonition>
-
-### Initialize `Estimator`
-
-Next, instantiate an [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator).
-
 ```python
-from qiskit.primitives import Estimator
+# observable(s) whose expected values you want to compute
+observable = SparsePauliOp(["II", "XX", "YY", "ZZ"], coeffs=[1, 1, -1, 1])
 
-estimator = Estimator()
+# value(s) for the circuit parameter(s)
+import numpy as np
+parameter_values = [[0], [np.pi/6], [np.pi/2]]
 ```
+### Initialize Estimator
```

Comment:
I would add a note on ISA conversion before splitting into the V1-V2 estimator workflows (note that I added an extra `python` at the end of the code cell, because the closing ``` were detected at the end of the suggestion, it should be removed).

```suggestion
The Qiskit Runtime primitives workflow requires circuits and observables to be transformed to only use instructions supported by the system (referred to as *instruction set architecture (ISA)* circuits and observables). The reference primitives still accept abstract instructions, as they rely on local statevector simulations, but transpiling the circuit might still be beneficial in terms of circuit optimization. 

```python
# Generate a pass manager without providing a backend
from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager

pm = generate_preset_pass_manager(optimization_level=1)
isa_circuit = pm.run(qc)
isa_observable = observable.apply_layout(isa_circuit.layout)
```python

### Initialize Estimator
```

Reply:
Addressed 596a5d854164e7ff566f095234a01bbafb41379c